setup.py

```diff
@@ -78,7 +78,7 @@
     "invenio-indexer>=1.1.0,<1.2.0",
     "invenio-jsonschemas>=1.0.0,<1.1.0",
     "invenio-pidstore>=1.1.0,<1.2.0",
-    "invenio-records-rest>=1.6.1,<1.7.0",
+    "invenio-records-rest>=1.6.2,<1.7.0",
```

Comment:
FYI, we are upgrading to `>=1.6.4` in [Niklas PR](https://github.com/inveniosoftware/invenio-app-ils/pull/618)